### ISSUE TITLE
"Сalling of moderator" / "Позвать модератора"

### ISSUE THREAD

**Issue by paul-k-pastvu** [PastVu/pastvu#503]
RU: Сейчас,  для любых обращений к модератору, служа комментарии (причем часто не к тем изображениям, к которым относятся). Также имеется постоянно действующая новостная тема https://pastvu.com/news/143 для уточнения точек съёмки, датировок и названий.

**Предложение:** 

1. Реализовать интерфейсный элемент (чекбокс? кнопка?) возле любого комментария, позволяющий пользователю привлечь внимание модератора (доступен любому зарегистрированному участнику, может быть отключен для участника в рамках ограничения доступа). По нажатию 
     1. рядом с комментарием появляется красный флажок "модератор вызван"
     2. генерируется событие специального типа (в рамках системы событий #239 ), которое записывается  в очередь для рассылки и может быть просмотрено в отдельном реестре
2. Для модераторов/админов доступна другая кнопка (?) которая позволяет отметить, что запрос был обработан. По нажатию 
     1. красный флажок рядом с комментарием  меняется на зеленую "птичку" - "запрос обработан"
     2. генерируется событие специального типа (в рамках системы событий #239), связанное с исходным

**Comment by yuriy-zhilovets2** [comment]
Конечно, нужна, потому что модератор может быть нужен по разным поводам. Например, для удаления дубля или для смены точки. Есть вариант выпадающего списка, но форма ввода все равно требуется для нестандартных случаев.
